platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/status/logic/ReplayingEventsStatusLogic.java

```diff
@@ -88,6 +89,23 @@ public PlatformStatusLogic processDoneReplayingEventsAction(@NonNull final DoneR
         }
     }
 
+    /**
+     * {@inheritDoc}
+     * <p>
+     * Receiving an {@link EmergencyReconnectStartedAction} while in {@link PlatformStatus#REPLAYING_EVENTS} throws an
+     * exception, since the check for potentially beginning an emergency reconnect doesn't happen until after
+     * events have been replayed.
```

Comment:
This is not quite right. When the recovery manifest file is detected, pre-consensus events should not be replayed at all even if present. In fact, any PCES files should be recycled once it is detected that an emergency state is required.

Reply:
I see your point - since PCES data is deleted prior to emergency recovery, nothing will be replayed.

With the consideration that the platform will pass through `REPLAYING_EVENTS` even if all PCES has been deleted, what do you think about this description instead?

```
     * Receiving an {@link EmergencyReconnectStartedAction} while in {@link PlatformStatus#REPLAYING_EVENTS} throws an
     * exception, since the check for potentially beginning an emergency reconnect doesn't happen until after
     * the platform has transitioned out of the {@link PlatformStatus#REPLAYING_EVENTS} status.
```